README.md

```diff
@@ -831,6 +831,23 @@ mrsk lock acquire -m "Doing maintanence"
 mrsk lock release
 ```
 
+## Gradual restarts
+
+When deploying to large numbers of hosts, you might prefer not to restart your services on every host at the same time.
+
+MRSK's default is to start new containers on all hosts in parallel. But you can control this by configuring `group_limit` and `group_wait`.
+
+```yaml
+service: myservice
+
+group_limit: 10
```

Comment:
Would like to be able to specify # of groups (or % of hosts) rather than fixed group size so the rolling restart size will scale up/down with host count. In Capistrano, we do 25% of hosts in each group. (Likely out of scope for this PR.)

Reply:
Good call. Could have the setting accept either format, like `10` or `25%`.

Reply:
Added in https://github.com/mrsked/mrsk/pull/222/commits/5ea2e20e51408fcfde4cc813cb44de70bbe002d9